cli/src/create.rs

```diff
@@ -85,10 +87,63 @@ pub fn create_project(arg_platform: &Option<String>) -> anyhow::Result<()> {
 }
 
 fn select_template() -> anyhow::Result<String> {
+    let (items, unselectable) = get_target_platforms_with_status()?;
+
     let idx = Select::with_theme(&ColorfulTheme::default())
         .with_prompt("Create template")
-        .items(&TEMPLATES)
-        .interact()?;
+        .items(&items)
+        .interact_on_opt(&Term::stderr())?;
+
+    if let Some(selected_idx) = idx {
+        if unselectable[selected_idx] {
+            style::print_yellow(format!(
+                "Cannot create {} template - build binding first",
+                &TEMPLATES[selected_idx]
+            ));
+            return select_template();
+        }
+        Ok(items[selected_idx].to_owned()) // Only available items will be matched with 'platform'
+    } else {
+        Err(Error::msg("Template selection failed"))
+    }
+}
+
+fn get_target_platforms_with_status() -> anyhow::Result<(Vec<String>, Vec<bool>)> {
+    let current_dir = env::current_dir()?;
+    let config = read_config(&current_dir.join("Config.toml"))?;
+
+    let mut items = Vec::new();
+    let mut unselectable = Vec::new();
+
+    for &template in TEMPLATES.iter() {
+        if template == "flutter" || template == "react-native" {
+            // Adding more information on the list
+            let requires = ["ios", "android"];
+            let missing: Vec<&str> = requires
+                .iter()
+                .filter(|&&req| !config.target_platforms.contains(req))
+                .cloned()
+                .collect();
+
+            if !missing.is_empty() {
+                items.push(format!(
+                    "{:<12} - Requires {} binding(s)",
+                    template,
+                    missing.join("/")
+                ));
+                unselectable.push(true);
+                continue;
+            }
+        }
+
+        if config.target_platforms.contains(template) {
+            items.push(template.to_string());
+            unselectable.push(false);
+        } else {
+            items.push(format!("{:<12} - Require binding", template));
+            unselectable.push(true);
+        }
```

Comment:
I found the correct logic is like this
You should display the result of `react-native` and `flutter` if it is selectable
```suggestion
            // continue;
            } else {
                items.push(template.to_string());
                unselectable.push(false);
            }
        } else if config.target_platforms.contains(template) {
            items.push(template.to_string());
            unselectable.push(false);
        } else {
            items.push(format!("{:<12} - Require binding", template));
            unselectable.push(true);
        }
```